apps/docs/src/app/core/component-docs/select/select-header/select-header.component.html

```diff
@@ -1,11 +1,7 @@
 <header>Select</header>
 <description>
-    The select component was created to emulate the native select element as closely as possible while also providing
+    The Select component is composed of the SelectComponent and the OptionComponent. It was created to emulate the native Select element as closely as possible while also providing
     more extended styling options.
```

Comment:
I moved the description of the component to the header (instead of the mode title) because you reference it in several descriptions. Can you confirm if it still makes sense?  Thanks :)